src/MICore/Transports/LocalLinuxTransport.cs

```diff
@@ -213,6 +197,42 @@ private void FifoWatcher_Deleted(object sender, FileSystemEventArgs e)
             }
         }
 
+        private bool GetRequiresRootAttach()
+        {
+            if (_miMode != MIMode.Clrdbg)
```

Comment:
The other check is on line 137 - I don't see that it was removed?  In any case, we probably only need one check or the other.